src/ServiceControl/CustomChecks/CustomCheckNotifications.cs

```diff
@@ -19,26 +19,27 @@ public void OnNext(IndexChangeNotification value)
         {
             try
             {
-                UpdateCount();
+                UpdateCount().GetAwaiter().GetResult();
```

Comment:
Is this signature async in later versions?

Reply:
Not sure though I figured it is better to align